group-coordinator/src/test/java/org/apache/kafka/coordinator/group/GroupMetadataManagerTest.java

```diff
@@ -5402,6 +5412,14 @@ private Map<Uuid, Set<Integer>> fromTopicPartitions(
         return assignmentMap;
     }
 
+    private Map<Integer, Set<String>> mkMapOfPartitionRacks(int numPartitions) {
```

Comment:
We have the same code in RecodHelperTests. Could we reuse it?

Reply:
yep makes sense, I'll make it public there and reuse